src/applications/vaos/routes.jsx

```diff
@@ -57,6 +57,12 @@ const ConfirmationPage = asyncLoader(() =>
 const ExpressCareList = asyncLoader(() =>
   import(/* webpackChunkName: "express-care" */ './components/ExpressCareList'),
 );
+const NewExpressCareRequestLayout = asyncLoader(() =>
+  import(/* webpackChunkName: "new-express-care-request-layout" */ './containers/NewExpressCareRequestLayout'),
```

Comment:
I think we should use `express-care` as the chunk name for both of these. That will group both of these components, and the new tab contents, into one separate bundle. Either that or have these two new additions be in an `express-care-request` bundle, and leave the tab as its own. I think we want to try to group all the pages on the form into one bundle, since we know someone going into the form is likely to use all of them.